GiftModels/Attachment.cs

```diff
@@ -36,5 +36,10 @@ public class Attachment
         /// catch all notes, description, w/e
         /// </summary>
         public string Description { get; set; }
+
+        /// <summary>
+        /// Whether or not an attachment is active, i.e., visible
+        /// </summary>
+        public bool IsActive { get; set; }

```

Comment:
Yes, and No.  I will ask them to update all the existing records to true.  The giving service will set this for all new records.